edx_proctoring/api.py

```diff
@@ -690,6 +690,20 @@ def create_exam_attempt(exam_id, user_id, taking_as_proctored=False):
             )
             log.error(log_msg)
 
+    if not external_id:
```

Comment:
I realize that there's similar logging a few lines down. But I'd like to get this log as close to the operation that generates the `external_id` as possible just to be absolutely sure that we wind up in a case with a falsy `external_id` here, because, we should be hitting the except clause for handling `BackendProviderSentNoAttemptID` if that's the case.

Reply:
To clarify, do practice exams come through with null `external_id`s for the exam or the exam attempt? If Proctrotrack has been returning null `external_ids` for the attempts, I would expect that we would see logging described in the exception handler for the `BackendProviderSentNoAttemptID` exception in Splunk, since there is no exception in that `register_exam_attempt` code for practice exams. I thought that the `external_id` here was referring to the `external_id` of the exam attempt and not the exam, because it's returned by the backend's `register_exam_attempt` method.